cogs/committee_actions_tracking.py

```diff
@@ -140,10 +140,9 @@ async def autocomplete_get_committee_members(
                 name=f"{member.display_name} ({member.global_name})", value=str(member.id)
             )
             for member in (
-                set(committee_role.members)
-                | set(
-                    committee_elect_role.members if committee_elect_role is not None else set()
-                )
+                set(committee_role.members) | set(committee_elect_role.members)
+                if committee_elect_role is not None
+                else set()
```

Comment:
Apologies. I forgot about the very low operator precedence of conditional expressions: https://docs.python.org/3/reference/expressions.html#operator-precedence
```suggestion
                set(committee_role.members) | (set(committee_elect_role.members)
                if committee_elect_role is not None
                else set())
```